qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -497,7 +503,7 @@ def test_stale_renew(self):
         self.mount_b.client_remote.run(args=["sudo", "kill", "-STOP", mount_b_pid])
 
         self.assert_session_state(mount_b_gid, "open")
-        time.sleep(self.mds_session_timeout * 1.5)  # Long enough for MDS to consider session stale
+        time.sleep(self.session_timeout * 1.5)  # Long enough for MDS to consider session stale
```

Comment:
should be just `session_timeout` -- this is causing failure here: http://pulpito.ceph.com/yuriw-2019-01-04_16:59:05-fs-wip-yuri2-testing-2019-01-03-2334-luminous-distro-basic-smithi/3421806/

Reply:
Thanks, fixed!